src/main/resources/application.properties

```diff
@@ -2,5 +2,5 @@ spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimezone=UTC
 spring.datasource.username=root
 spring.datasource.password=dima1234
 
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
It's a bad practice to use both liquibase and hibernate as sources for DB structure.
Please return back to validate strategy